<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralWidget
{
    background-image: url(:/doge.png);
}</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhHiddenText</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="accessibleName">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>931</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Tarttetko nää massia? Täältä sitä saa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLueKortti">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>380</y>
      <width>281</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Reference Sans Serif&quot;;

background-color: rgba(170, 170, 0, 225);
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>LUE KORTTI</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>921</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans</family>
      <pointsize>18</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 63 18pt &quot;Lucida Sans&quot;;

color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>SYÖTÄ PANKKIKORTTI</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Sans Unicode</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 8pt &quot;Lucida Sans Unicode&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>(syötä pankkikortin ID-numero)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditKortti">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(170, 170, 117, 230);
border-color: rgb(0, 21, 0);
gridline-color: rgb(0, 22, 0);

font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>520</y>
      <width>811</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>*formlayout on karkeaversio toiminnallisuuksien testausta varten*</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>631</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 16pt &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>Virhe sisäänkirjautumisessa!
   Tarkista syötetty kortti</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>721</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>	VÄÄRÄ PIN-KOODI

Pin-koodi syötetty väärin liian monta kertaa
  KorttiID# on poistettu käytöstä</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>lineEditKortti</zorder>
   <zorder>btnLueKortti</zorder>
   <zorder>label_6</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
